src/layout/Dropdown/DropdownComponent.tsx

```diff
@@ -14,9 +14,18 @@ import type { PropsFromGenericComponent } from 'src/layout';
 export type IDropdownProps = PropsFromGenericComponent<'Dropdown'>;
 
 export function DropdownComponent({ node, formData, handleDataChange, isValid, overrideDisplay }: IDropdownProps) {
-  const { optionsId, preselectedOptionIndex, id, readOnly, mapping, source, textResourceBindings } = node.item;
+  const {
+    optionsId,
+    options: staticOptions,
+    preselectedOptionIndex,
+    id,
+    readOnly,
+    mapping,
+    source,
+    textResourceBindings,
+  } = node.item;
   const { langAsString } = useLanguage();
-  const options = useGetOptions({ optionsId, mapping, source })?.filter(duplicateOptionFilter);
+  const options = useGetOptions({ optionsId, mapping, source })?.filter(duplicateOptionFilter) || staticOptions;
```

Comment:
FYI, the Select component from the design system throws an exception if there are duplicated option values, which is why duplicates are filtered out from `useGetOptions`.

Reply:
Oh.. 🤔 But I guess that's fair? 🤷 Errors in the input should not be silenced, but should throw errors, IMO - although throwing exceptions is not that developer-friendly, I guess.

If I was developing an app with 4 options and only saw 3 of them, I could end up spending way too much time trying to figure out why one of the options 'disappeared', instead of immediately noticing that there are duplicate values (if I just saw the exception with an explanation in the console).

Reply:
It might be a breaking change though

Reply:
Right - fixing it properly in `useGetOptions()` now, in such a way that others can't easily do the same mistake.